### DISCUSSION TITLE
What is plot status "?:?"  I've got a plot in that phase for quite a long time.  Also what is "wall" in the timers?

### DISCUSSION THREAD

**Discussion by DroptheHammer** [ericaltendorf/plotman#581] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/79561845/118857521-fa58fc00-b88c-11eb-9102-d0530b2230d7.png)

Thanks,
DTH

**Comment by calarndt** [comment, marked as answer]
Not sure about wall. However plot id of --------- and ?:? indicate to me you have a plot running somewhere that was not started by plotman and has no log in your logs dir.

**Comment by cyperbg** [comment]
wall is how long the plot job has been running for.

**Reply by altendky** [reply]
As in 'how much time has passed on the clock on the wall since the process started'.  'wall clock' time, as opposed to usr and sys time that can count faster than a wall clock for example due to multiple threads running in the process.
